src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue23774.cs

```diff
@@ -0,0 +1,26 @@
+#if !MACCATALYST
+using NUnit.Framework;
+using NUnit.Framework.Legacy;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+    public class Issue23774 : _IssuesUITest 
+	{
+        public override string Issue => "[Android] Text Clipping Issue in Entry with Long Strings";
+
+        public Issue23774(TestDevice device) : base(device)
+        {
+        }
+
+        [Test]
+		[Category(UITestCategories.Entry)]
+		public void VerifyEntryIsReadOnlyInitialLoading()
+        {
+			App.WaitForElement("Label");
+			VerifyScreenshot();
```

Comment:
Test snapshots already available on the latest PR build.

Reply:
I’ve added snapshots for all platforms